modules/graphics/demo-apps.nix

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}: let
+  cfg = config.ghaf.graphics.demo-apps;
+  weston = config.ghaf.graphics.weston;
+
+  /* Scaled down firefox icon */
```

Comment:
I don't want to be too much of a style police, but this is the only place in this project where C-style comments are used, I actually didin't know before this Nix supports such comment style. Elsewhere the commenting style with `#` is used

Reply:
https://github.com/tiiuae/ghaf/blob/main/modules/graphics/weston.ini.nix#L20 here also C-style comments, actually I split this file, and tried to keep same style.